common/openapi.yml

```diff
@@ -543,12 +543,14 @@ components:
             $ref: '#/components/schemas/JobReport'
 
     GetPipelinesResult:
-      type: array
-      items:
-        $ref: '#/components/schemas/Pipeline'
+      description: result of a getPipelines request
+      type: object
```

Comment:
changing from object to array will allow this response to be processed properly by pydantic, we think